business_requirement_activity_report/README.rst

```diff
@@ -2,9 +2,9 @@
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
 
-=========================
-Timesheet Activity Report
-=========================
+====================================
+Business Requirement Activity Report
```

Comment:
why the name change to Business Requirement Activity Report , so it should be in the OCA business requirement repo ?